src/backend/services/file/service.py

```diff
@@ -1,95 +1,56 @@
-import os
-import re
-from pathlib import Path
+import io
 from typing import Any
 
+import pandas as pd
+from fastapi import UploadFile as FastAPIUploadFile
+from pypdf import PdfReader
 
-class FileService:
-    DEFAULT_DATA_FOLDER = "src/backend/data"
+PDF_EXTENTION = "pdf"
+TEXT_EXTENTION = "txt"
+MARKDOWN_EXTENTION = "md"
+CSV_EXTENTION = "csv"
+EXCEL_EXTENTION = "xlsx"
+EXCEL_OLD_EXTENTION = "xls"
+JSON_EXTENTION = "json"
+DOCX_EXTENTION = "docx"
 
-    def __init__(self):
-        current_directory = Path(Path.cwd())
-        data_folder = Path(self.DEFAULT_DATA_FOLDER)
 
-        self.folder_path = current_directory.joinpath(data_folder)
+def get_file_extention(file_name: str) -> str:
+    return file_name.split(".")[-1].lower()
 
-    def create_file_folder(self):
-        self.folder_path.mkdir(exist_ok=True)
 
-    def generate_new_filepath(self, file_path: Path) -> Path:
-        """
-        Generate a new file path. The path is generated by concatenating the base name with a counter index.
+async def get_file_content(file: FastAPIUploadFile) -> str:
+    file_contents = await file.read()
+    file_extention = get_file_extention(file.filename)
 
-        Args:
-            file_path (Path): File path.
+    if file_extention == PDF_EXTENTION:
```

Comment:
Also nit to just change the wording from `extention` to `extension` where applicable